src/main/java/jpo/sdw/depositor/DepositorProperties.java

```diff
@@ -12,6 +10,8 @@
 import org.springframework.context.annotation.PropertySource;
 import org.springframework.core.env.Environment;
 
+import jakarta.annotation.PostConstruct;
```

Comment:
Is this annotation used?

Reply:
The `initialize()` method in this class is annotated using this annotation.